src/components/AddForm/types.ts

```diff
@@ -0,0 +1,16 @@
+import {WithStyles} from "@material-ui/core";
+import {styles} from "./style";
+
+export type Course = {
+    id: number;
+    course: string;
+    name: string;
+    img: string;
+    author: string;
+    stars: number;
+    price: number;
+    info: string;
+};
+export interface Props extends WithStyles <typeof styles> {
```

Comment:
If you'll remove styles file you need to remove `extends WithStyles<typeof style>`